### DISCUSSION TITLE
can I turn the computer off so I can add extra HDD without impacting farming?

### DISCUSSION THREAD

**Discussion by Chia-Forever** [Chia-Network/chia-blockchain#4786] [Support]
can I turn the computer off so I can add extra HDD without impacting farming?

**Comment by lvcivs** [comment]
Your computer will not be farming while it is turned off, I guess that is rather obvious.

**Reply by Chia-Forever** [reply]
thanks Ivcivs, would it be easy to go back to farming once the computer is on again, and would it be automatic? or do I need to do anything?

**Reply by lvcivs** [reply]
Yes, it will sync automatically. Should be pretty quick, usually just a few minutes to "catch up" with the network. But depends on how long your machine was offline, of course.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
